### PR TITLE
Prompted the config if not set, moved all constants to util constants.go

### PR DESCRIPTION
**Pull request by AnirudhPokala**
* Created ConfigCmdCreateRun() function in config.go (pmk).
* Prompted the config if not set (during check-node/ prep-node) 
* Moved the constants in cmd/constants.go to pkg/util/constants.go 
Screenshot 2021-02-25 at 1 46 04 PM

### CODE REVIEW

File: pkg/util/constants.go [hunk 1]
```diff
@@ -47,9 +48,28 @@ var (
...
 
 func init() {
 	RequiredPorts = []string{"443", "2379", "2380", "8285", "10250", "10255", "4194", "8285"}
 	PortErr = fmt.Sprintf("Ports required to be available %s", RequiredPorts)
 }
+
+//This is the current version of cli.
+const (
+	Version string = "pf9ctl version: v1.0"
+)
```
Comment (anupbarve) [inline pkg/util/constants.go:75]: Why is this visible in the current diff ?
Can you rebase and repost ?

Comment (AnirudhPokala) [reply pkg/util/constants.go:75]: we have moved all variables and constants from cmd/constants.go to pkg/util/constants.go right.

File: pkg/pmk/config.go [hunk 2]
```diff
@@ -23,6 +28,23 @@ type Config struct {
...
+	Pf9DBDir = filepath.Join(Pf9Dir, "db")
+	// Pf9DBLoc represents location of the config file.
+	Pf9DBLoc = filepath.Join(Pf9DBDir, "config.json")
+	// Pf9Log represents location of the log.
+	Pf9Log = filepath.Join(Pf9LogDir, "pf9ctl.log")
+	// WaitPeriod is the sleep period for the cli
+	// before it starts with the operations.
+	WaitPeriod = time.Duration(60)
+)*/
+
```
Comment (anupbarve) [inline pkg/pmk/config.go:47]: Remove.

File: pkg/pmk/config.go [hunk 3]
```diff
@@ -47,7 +70,10 @@ func LoadConfig(loc string) (Config, error) {
 	if err != nil {
 
 		if os.IsNotExist(err) {
-			return Config{}, ErrConfigurationDetailsNotProvided
+			// to initiate the config create and store it
```
Comment (anupbarve) [inline pkg/pmk/config.go:73]: Could you please add a console log saying that existing config not found, prompting for new config.

**Review by anupbarve — CHANGES_REQUESTED** [review]

File: pkg/pmk/config.go (hunk 3 above)
Comment (anupbarve) [reply pkg/pmk/config.go:56]: Hey @AnirudhPokala , 
I cannot see this change in new commit.

**Review by anupbarve — APPROVED** [review]

**Pull request merged**